source/cloud/cloud-integrations/integrations-faq.md

```diff
@@ -89,7 +89,7 @@ Deployments that cannot create bot accounts via webhooks due to security reasons
      
      The bot should retrieve the session token from the `Token` header and store it in memory for use with future requests.
    
-   **Note:** Each session token has an expiry time, set depending on the server's configuration. If the session token your bot is using expires, it will receive a `401 Unauthorized` response from requests using that token. When your bot receives this response, it should reapply the login logic (using the above steps) to get another session token. Then resend the request that received the `401` status code.
+   **Note:** Each session token has an expiry time, set depending on configuration. If the session token your bot is using expires, it will receive a `401 Unauthorized` response from requests using that token. When your bot receives this response, it should reapply the login logic (using the above steps) to get another session token. Then resend the request that received the `401` status code.
```

Comment:
```suggestion
   **Note:** Each session token has an expiry time, set depending on configuration. If the session token your bot is using expires, it will receive a `401 Unauthorized` response from requests using that token. When your bot receives this response, it should reapply the login logic (using the above steps) to get another session token. Then it should resend the request that received the `401` status code.
```